README.md

```diff
@@ -1,65 +1,49 @@
 # php-selenium-browserstack
+Code samples for php selenium tests on browserstack.
 
 ## Prerequisites 
-1. `php` and `composer` should be installed in your system
-2. Go to the directory where you have your test scripts and run the command below:
+1. `php` and `composer` should be installed in your system.
+* If you don't already use Composer, you can download the composer.phar binary:
 ```
-  composer require php-webdriver/webdriver
```

Comment:
Where are we doing this step now?

Reply:
`php composer.phar install` takes care of it